R/report_card.R

```diff
@@ -693,6 +693,44 @@ extract_rc_enrollment <- function(list_of_prs, cds_identifiers = TRUE) {
       )
   }
   
-  out
+  out %>%
+    rename(county_id = county_code,
+           district_id = district_code, 
+           school_id = school_code) %>%
+    select(-school_name) %>%
+    return()
+
 }
 
+
+#' Enrich report card subgroup percentages with best guesses at 
+#' subgroup numbers
+#' 
+#' @param df data frame of icluding subgroup percentages
+#' 
+#' return data_frame
+#' @export
+enrich_rc_enrollment <- function(df) {
```

Comment:
this is clever - so to restate, we need this because special_pop data doesn't contain the number of students, just percentages, so to build the aggregates we need to pull them from `extract_rc_enrollment()`?

Reply:
yes, exactly! 

looking back, #134 includes the extracting/enriching enrollment counts directly into `fetch_special_pop`. i think that reduces the number of times user has to wait for the rc file download...

Reply:
#142 addresses this